docs/releases/1.10.txt

```diff
@@ -471,6 +471,31 @@ creates the possibility of a deadlock. To adapt your code in the case of RQ,
 you can `provide your own worker script <http://python-rq.org/docs/workers/>`_
 that calls ``django.setup()``.
 
+Timezone-aware :class:`~django.core.files.storage.Storage` API
```

Comment:
You put this in "Backwards-incompatible changes" but shouldn't it be more like "new features"? For clarify of what I changed, I'll leave it here in my edits.